src/main.rs

```diff
@@ -55,6 +55,12 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .help("the nu script to run")
                 .index(1),
         )
+        .arg(
+            Arg::with_name("args")
+                .help("positional args (used by --testbin)")
+                .index(2)
+                .multiple(true),
+        )
```

Comment:
Not a huge fan of this, but couldn't think of a nicer way to do this right now.

Maybe it would be better for us to move towards subcommands?

```
nu testbin eval <command strings>
nu testbin run <script>
nu testbin <name> <args>
```